### PR TITLE
Tarask/debug/gh pages status check fail

### PR DESCRIPTION
**Pull request by fog1985**
## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [x] tested locally
- [x] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

**Review by exu — APPROVED** [review]
LGTM

File: .github/workflows/helm-releaser-all-charts.yaml [hunk 1]
```diff
@@ -96,11 +96,9 @@ jobs:
               echo "Checking latest GH pages build status --> $status"
               sleep 5
 
-              # checking if GH pages built doesn't take too much time or it didn't go into erroring state.
-              # waiting time for build to succeed is set to 10 minutes.
               counter=$(expr $counter + 1)
-              if [[ $status =~ error.* ]] || [[ $counter == 120 ]]; then
-                echo Something went wrong. Please check GH's pages issues.
+              if [[ $status == \"errored\" ]] || [[ $counter == 120 ]]; then
```
Comment (nicufk) [inline .github/workflows/helm-releaser-all-charts.yaml:100]: typo? or there is the string with "errored"?

**Review by nicufk — APPROVED** [review]

File: .github/workflows/helm-releaser-all-charts.yaml (hunk 1 above)
Comment (fog1985) [reply .github/workflows/helm-releaser-all-charts.yaml:100]: Yeap. it says not an error but errored.

Comment (fog1985) [reply .github/workflows/helm-releaser-all-charts.yaml:100]: You can check here: https://github.com/kubeshop/helm-charts/runs/4186834233?check_suite_focus=true

**Pull request merged**